src/hooks/useStore.js

```diff
@@ -59,51 +61,83 @@ const useStore = create(
 					};
 				});
 			},
-	addTransaction: (accountId, transaction) => {
-		set(state => {
-			const transactionId = nanoid();
-			const {accounts} = get();
-			const index = accounts.findIndex(account => account.id === accountId);
-			accounts[index].transactions.push(transactionId);
-			const index2 = categories.findIndex(category => category.id === transaction.categoryId);
-			categories[index2].transactions.push(transactionId);
-			return {
-				accounts: [...accounts],
-				transactions: [
-					...state.transactions,
-					{
-						id: transactionId,
-						accountID: accountId,
-						...transaction,
-					},
-				],
-			};
-		});
-	},
-      editTransaction(id, update) {
-        set(state => {
-          const transactions = state.transactions.map(transaction => {
-            return transaction.id === id ? {...update, id} : transaction;
-          });
-          const categories = state.categories.map(category => {
-            const index = category.transactions.indexOf(id);
-            if (index > -1) {
-              category.transactions.splice(index, 1);
-            }
-            if (category.id === update.categoryId) {
-              category.transactions.push(id);
-            }
-            return category;
-          });
-          return {transactions, categories};
-        });
-      },
+			addTransaction: (accountId, transaction) => {
+				set(state => {
+					const transactionId = nanoid();
+					const {accounts} = get();
+					const index = accounts.findIndex(account => account.id === accountId);
+					accounts[index].transactions.push(transactionId);
+					accounts[index].value += transaction.change;
+					accounts[index].value = Math.round(accounts[index].value * 100) / 100;
+					const index2 = categories.findIndex(
+						category => category.id === transaction.categoryId
+					);
+					categories[index2].transactions.push(transactionId);
+					return {
+						accounts: [...accounts],
+						transactions: [
+							...state.transactions,
+							{
+								id: transactionId,
+								accountID: accountId,
+								...transaction,
+							},
+						],
+					};
+				});
```

Comment:
A general think I noticed in your code is the lack of line breaks. This often improves readability:
```suggestion
					const transactionId = nanoid();
					const {accounts} = get();
					const index = accounts.findIndex(account => account.id === accountId);
					
					accounts[index].transactions.push(transactionId);
					accounts[index].value += transaction.change;
					accounts[index].value = Math.round(accounts[index].value * 100) / 100;
					
					const index2 = categories.findIndex(
						category => category.id === transaction.categoryId
					);
					categories[index2].transactions.push(transactionId);
					
					return {
						accounts: [...accounts],
						transactions: [
							...state.transactions,
							{
								id: transactionId,
								accountID: accountId,
								...transaction,
							},
						],
					};
				});
```